src/tests/test_dynamic_api/test_serializers.py

```diff
@@ -507,3 +508,36 @@ def test_no_display_title_present(
         assert "'title': 'reference for DISPLAY FIELD'" not in str(
             fietsplaatjes_serializer.data
         )
+
+    @staticmethod
+    def test_uri_field_present(explosieven_model):
+        """ Prove that a URLfield is stored as a charfield and default 200 in length """
+        assert (
+            explosieven_model._meta.get_field("pdf").get_internal_type() == "CharField"
+        )
+        assert explosieven_model._meta.get_field("pdf").max_length == 200
+
+    @staticmethod
+    def test_uri_field_can_validate(
+        api_request, explosieven_schema, explosieven_model, explosieven_data
+    ):
+        """ Prove that a URLfield can be validated by the URIValidator """
+
+        ExplosievenSerializer = serializer_factory(explosieven_model, 0, flat=True)
+
+        api_request.dataset = explosieven_schema
+
+        validate_uri = URLValidator()
+
+        explosieven_serializer = ExplosievenSerializer(
+            explosieven_data, context={"request": api_request}
+        )
+
+        try:
```

Comment:
Het is beter om `try` - `except` statements in tests niet te gebruiken bij het testen van de "happy flow". Lat maar ValidationError komen als het niet werwacht is.

Reply:
Ok check